packages/app-accounts/src/Overview.tsx

```diff
@@ -26,10 +26,16 @@ type State = {
 };
 
 class Overview extends React.PureComponent<Props, State> {
-  state: State = {
-    isCreateOpen: false,
-    isImportOpen: false
-  };
+  constructor (props: Props) {
+    super(props);
+    const { state = {} } = this.props.location;
```

Comment:
(nit) Blank line before.

Then, can do this as `const { state: { isCreateOpen } = {} } = props;`

(Removing the line below)

Reply:
I guess `const { state : { isCreateOpen = false } = {} } = this.props.location;` works (and avoids the need to `check isCreateOpen` for undefined. Your suggestion bothered TS because `isCreateOpen` wasn't initialized.

Reply:
The below comment would have fixed that, i.e. `!!isCreateOpen`, however the `= false` also works.

Reply:
TS was complaining at this exact line `const { state: { isCreateOpen } = {} } = props` so that I had to set a default here.